tools/rawtools/rawtools.xml

```diff
@@ -29,7 +29,14 @@
         ## TODO: To remove this patch when dropping support for Galaxy 17_09, adding profile="18.01" to the tool.
         export HOME=\$cwd  &&
 
-        ln -s '$input_raw' input.raw &&
+        mkdir raws_folder &&
+        #for $input_raw in $input_raws:
+            #set $input_name = $input_raw.element_identifier.split('/')[-1].replace(".raw", "") + ".raw"
```

Comment:
You probably want to use something like that as well:

`#set $bam_name = re.sub('[^\w\-\.]', '_', str($bam.element_identifier))` to get rid of spaces and so on ...